src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -20,5 +21,28 @@ public IObservable<IPullRequestModel> CreatePullRequest(IRepositoryHost host, IS
 
             return host.ModelService.CreatePullRequest(repository, title, body, source, target);
         }
+
+        public string GetPullRequestTemplate(ISimpleRepositoryModel repository)
+        {
+            Extensions.Guard.ArgumentNotNull(repository, nameof(repository));
+
+            var paths = new[]
+            {
+                Path.Combine(repository.LocalPath, "PULL_REQUEST_TEMPLATE"),
+                Path.Combine(repository.LocalPath, "PULL_REQUEST_TEMPLATE.md"),
+                Path.Combine(repository.LocalPath, ".github", "PULL_REQUEST_TEMPLATE"),
+                Path.Combine(repository.LocalPath, ".github", "PULL_REQUEST_TEMPLATE.md"),
```

Comment:
We could just keep the strings in an static array (without the path), and only do the Path.Combine when we're actually checking for it down below as we need it, and order the strings for likelihood (we recommend a `PULL_REQUEST_TEMPLATE.md` so we can check for that first). That way we only allocate once and we're likely to only do a single Path.Combine operation (not that it's very costly, but kinda wasteful as-is)